options_test.go

```diff
@@ -25,6 +25,108 @@ import (
 	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
 )
 
+func TestGetFirecrackerConfig(t *testing.T) {
+	tempFile, err := ioutil.TempFile("", "firectl-test-drive-path")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.Remove(tempFile.Name())
```

Comment:
minor: I'd probably put this in a single defer just to make order explicit.  I always have to look up what gets called first when there are multiple.

```go
defer func() {
	tempFile.Close()
	os.Remove(tempFile.Name())
}
```